### PR TITLE
Print binary version info on opa-openshift startup

### PR DESCRIPTION
**Pull request by rubenvp8510**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -45,7 +46,7 @@ deps: go.mod go.sum
 	go mod verify
 
 opa-openshift: deps main.go $(wildcard *.go) $(wildcard */*.go)
-	CGO_ENABLED=0 GOOS=$(OS) GOARCH=amd64 GO111MODULE=on GOPROXY=https://proxy.golang.org go build -a -ldflags '-s -w' -o $@ .
+	CGO_ENABLED=0 GOOS=$(OS) GOARCH=amd64 GO111MODULE=on GOPROXY=https://proxy.golang.org go build -a -ldflags '-s -w -X main.Version=$(VERSION) -X main.Branch=$(GIT_BRANCH)' -o $@ .
```
Comment (periklis) [inline Makefile:49]: Missing `-X main.Revision` here?

**Comment by periklis** [comment]
@rubenvp8510 Since we add this nice way to package the version with the binary, I suggest to drop by default a log line when some starts this container anyway? WDYT?

File: Makefile (hunk 1 above)
Comment (rubenvp8510) [reply Makefile:49]: yep, nice catch I'll add it.

**Comment by rubenvp8510** [comment]
> @rubenvp8510 Since we add this nice way to package the version with the binary, I suggest to drop by default a log line when some starts this container anyway? WDYT?

I think that will be a good idea! I'll update the PR with that change.

**Comment by rubenvp8510** [comment]
@periklis Thanks for the review, I've already added the changes requested. Please give it another review when you get a chance.

**Comment by rubenvp8510** [comment]
@periklis Can you also review this one please? I did the same changes we agreed on the api gateway.

**Review by periklis — APPROVED** [review]

**Pull request merged**